CMakeLists.txt

```diff
@@ -45,31 +49,47 @@ if(NOT GENESIS_BUILD_TOOLS AND GENESIS_AUTOFORMAT)
   message(FATAL_ERROR "Cannot enable GENESIS_AUTOFORMAT without GENESIS_BUILD_TOOLS")
 endif()
 
-if(GENESIS_BUILD_EDITOR)
-    set(GENESIS_BUILD_SHADERS ON)
+if(GENESIS_ENABLE_EDITOR)
+  set(GENESIS_BUILD_RUNTIME ON) # Editor requires the runtime lib
```

Comment:
This isn't going to work, `set(X y)` will not overwrite an already existing cache variable `X`.

Reply:
I think my new changes SHOULD work but you will likely need to double check my work.